scripts/circleci_auto_approve.py

```
@@ -0,0 +1,198 @@
+  for check in check_runs(config):
+    if check.get("name") != expected_check_name:
+      continue
+    workflow_id = circleci_workflow_id_from_check(check)
+    if workflow_id:
+      return workflow_id
+  return None
+
+
+def find_approval_request(config: Config, workflow_id: str) -> tuple[str, str] | None:
+  url = f"{CIRCLECI_API}/workflow/{workflow_id}/job"
+  payload = request_json(url, token=config.circleci_token, token_header="Circle-Token")
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "token_header"

(S106)